src/main/java/net/tropicraft/core/common/entity/damage/DartDamage.java

```diff
@@ -0,0 +1,12 @@
+package net.tropicraft.core.common.entity.damage;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.util.EntityDamageSourceIndirect;
+
+public class DartDamage extends EntityDamageSourceIndirect {
```

Comment:
What's the point of this class?

Reply:
oh this was used when I had darts then I removed them...my b